src/startup.rs

```diff
@@ -85,14 +84,18 @@ async fn run(
             ))
             .wrap(TracingLogger::default())
             .service(
-                web::scope("/admin")
-                    .wrap(from_fn(reject_anonymous_users))
-                    .route("/dashboard", web::get().to(admin_dashboard))
-                    .route("/password", web::get().to(change_password_form))
-                    .route("/password", web::post().to(change_password))
-                    .route("/logout", web::post().to(log_out)),
+                web::scope("/api")
+                    .service(
+                        web::scope("/rooms")
+                            .wrap(from_fn(reject_anonymous_users))
+                            .route("/", web::get().to(health_check)),
+                    )
+                    .service(
+                        web::scope("/auth")
```

Comment:
Should we rename "/auth" to "/authentication" because people confuse the two "auths" all the time. ("authn"/"authentication" and "authz"/"authorization"). Btw I'm trying the suggestion feature in GitHub, hope it works.
```suggestion
                        web::scope("/authentication")
```